eta/core/video.py

```diff
@@ -2368,7 +2368,7 @@ def _retrieve(self):
 
         width, height = self.frame_size
         try:
-            vec = np.fromstring(self._raw_frame, dtype="uint8")
+            vec = np.frombuffer(self._raw_frame, dtype="uint8")
```

Comment:
Noting that the array output by `_retrieve()` will now be read-only with this change.